### ISSUE TITLE
colorCustomizations Toggle

### ISSUE THREAD

**Issue by dylan-k** [tcg/vscode-toggle-settings#16]
I want to toggle the appearance of those squiggly lines that under the errors as I type in VSCode. I tried the following in my settings file, with the plugin installed, but the command does nothing.
```json
"toggleSettingsChanges.settingsToToggle": {
        "workbench.colorCustomizations": {
          "editorError.foreground": "#00000000",
          "editorWarning.foreground": "#00000000",
          "editorInfo.foreground": "#00000000"
        }
},
```

**Comment by tcg** [comment]
@dylan-k I believe you, but I am not able to reproduce this currently. 

If you run the command from the Command Palette, does it *not* create/update a file in `[working-dir]/.vscode/settings.json`? 

<img width="593" alt="image" src="https://github.com/tcg/vscode-toggle-settings/assets/575773/3e7595a7-f891-4bf3-9bcf-7fc7f26c1751">

**Comment by dylan-k** [comment]
That's right, there are no changes, when I run the command to the file at `[working-dir]/.vscode/settings.json`

**Comment by tcg** [comment]
@dylan-k I'm really sorry to hear that. 

I'm not yet able to reproduce that behavior. If you're feeling adventurous, you can try looking for error output from extensions in VSCode.
